mozilla_vpn/dashboards/vpn_saasboard_churn.dashboard.lookml

```diff
@@ -41,33 +40,40 @@
   - name: " (3)"
     type: text
     title_text: ''
-    body_text: |2-
+    subtitle_text: ''
+    body_text: |2
 
 
       <div style="border-top: solid 2px #e0e0e0;">
 
       <h3><b>Churn Rate</b></h3>
+
+      <h4> <b><code>Original Subscription Start Date</code> is used to discard subscription removal due to plan changes(upgrades).</b> </h4>
     row: 8
     col: 0
     width: 24
-    height: 2
+    height: 3
   - title: Churn by Months Since Subscription Start
     name: Churn by Months Since Subscription Start
     model: mozilla_vpn
     explore: subscriptions
     type: looker_column
-    fields: [subscriptions__retention.months_since_subscription_start, subscriptions__retention.churned,
-      subscriptions__retention.previously_retained]
+    fields: [original_subscriptions__retention.churned, original_subscriptions__retention.previously_retained,
+      original_subscriptions__retention.months_since_original_subscription_start]
     filters:
-      subscriptions__retention.months_since_subscription_start: ">0"
-      subscriptions__retention.is_cohort_complete: 'Yes'
-    sorts: [subscriptions__retention.months_since_subscription_start]
+      original_subscriptions__retention.is_cohort_complete: 'Yes'
+      original_subscriptions__retention.months_since_original_subscription_start: ">0"
+    sorts: [original_subscriptions__retention.months_since_original_subscription_start]
     dynamic_fields: [{category: table_calculation, expression: "${subscriptions__retention.churned}/${subscriptions__retention.previously_retained}",
```

Comment:
There are a bunch of such `dynamic_fields` expressions in these dashboards that seem like they might need to be updated to use `original_subscriptions__retention` rather than `subscriptions__retention`.

Reply:
I looked up where these `dynamic_fields` expressions defined and what they seem to do as the dashboard lookML is automatically generated after editing the dashboard using the GUI. It contains the table calculations in each tile (customized explore). For this particular part, it is a churn calculation for churn by plan.

Reply:
But it seems like the dynamic fields that still reference `subscriptions__retention` will be calculating things differently than intended.

With most other references to `subscriptions__retention` being replaced with `original_subscriptions__retention` I'm guessing the dynamic fields should also be changed.

Reply:
I am not sure if I understand what you are referencing correctly. For example, in ln 67 to on 76 for the new version, there are two calculations labeled `Churn Rate` and `Total Churn Rate`. The first one uses `subscriptions__retention` and the second one uses `original_subscriptions__retention`. For the dashboard plot, `Total Churn Rate` is plotted and `subscriptions__retention` is included in the explore for users to see the difference in the calculations. 
Is this what you noticed or am I missing your point?

Reply:
OK, I see what you mean for this one.

There are some other charts that where the dynamic fields only reference `subscriptions__retention` and not `original_subscriptions__retention` ("_Churn Counts Table (by Cohort)_" and all the "_(by Plan Interval Type)_" ones), but it sounds like you've thought these things through. 👍